dockerfiles/init/manifests/che.env

```diff
@@ -122,14 +122,24 @@
 #     agents within the workspace when it boots, then the workspace will not be started.
 #CHE_WORKSPACE_AGENT_DEV_MAX__START__TIME__MS=300000
 
-# Workspace Agent Java Options
-#   Java command line options to be added to JVM's that run agents within workspaces.
-#CHE_WORKSPACE_JAVA_OPTIONS=-Xms256m -Xmx2048m -Djava.security.egd=file:/dev/./urandom
+
+
+# Workspace Java Options
+#   Java command line options to be added to JVM's that running within workspaces.
+#CHE_WORKSPACE_JAVA__OPTIONS=NULL
```

Comment:
Usually che.env has default value from che.properties. Is null value better here?

Reply:
strongly agree, it won't be used by default but will give a ray of lite for users what actually default is.